SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005861	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Maximum extinction count: #{CaptionRatio}#
+SKILL_20151224_005862	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Strike with powerful hand knife to damage the enemy.
+SKILL_20151224_005863	Maximum purchase count: #{CaptionRatio}#
+SKILL_20151224_005864	Decatose
+SKILL_20151224_005865	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Attack with the collected money in order to offer to the gods. Enemies, but boss can be instantly killed at 1/10 rate and the money from instant kill can be picked back up.
+SKILL_20151224_005866	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Silver consumption: #{CaptionRatio}#
+SKILL_20151224_005867	Duration: #{CaptionRatio}#second{nl}Master level 1
+SKILL_20151224_005868	{#993399}{ol}[Magical] - [Holy]{/}{/}{nl}Temporarily creates magic circle which grants hyper regeneration. If you or your comrades are damaged by an enemy attack inside of the magic circle, HP will be quickly recovered.
+SKILL_20151224_005869	Duration: #{CaptionTime}#second{nl}HP Recovery: #{CaptionRatio}#
+SKILL_20151224_005870	{#993399}{ol}[Magical] - [Fire]{/}{/}{nl}Incinerate enemies with status ailments. The duration of incineration will increase depending on the amount of status ailments.
+SKILL_20151224_005871	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Basic duration: #{CaptionRatio2}#second
+SKILL_20151224_005872	Intentionally let you or your comrades bleed to prevent further status ailments. This state will be invulnerable to all rank 2 or lower status ailments.
+SKILL_20151224_005873	Duration: #{CaptionTime}#second{nl}HP Reduction: #{CaptionRatio}#
+SKILL_20151224_005874	Spray a cloud of antidote that cures your comrade's status ailments. The comrades within the effect will be cured of rank 3 or lower status ailments.
+SKILL_20151224_005875	Remove rank 3 and lower status effect{nl}Maximum application count: #{CaptionRatio}#
+SKILL_20151224_005876	Spread all status ailments on the enemies to other enemies within the range.
+SKILL_20151224_005877	Maximum application count: #{CaptionRatio}#
+SKILL_20151224_005878	Equip the beak mask. The front side of the mask has special drug applied which can block off rank 2 or lower status ailments.
+SKILL_20151224_005879	Duration: #{CaptionTime}#second{nl}Rank 1 and 2 immunity: 100%{nl}Rank 3 immunity: 80%
+SKILL_20151224_005880	{#993399}{ol}[Magical] - [Holy]{/}{/}{nl}Uses the power of goddess to revenge the with x7 the strength of their attack.
+SKILL_20151224_005881	Duration: #{CaptionTime}#second{nl}Additional damage: #{CaptionRatio}#%
+SKILL_20151224_005882	{#993399}{ol}[Magical] - [Holy]{/}{/}
+SKILL_20151224_005883	{#993399}{ol}[Magical] - [Holy]{/}{/}{nl}Summon a magic circle which drastically increases maximum HP for a period of time.
```

Comment:
[Magical] -> [Magic]

Set up a magic circle that drastically increases your maximum HP for a period of time.